shopno/index.html

```diff
@@ -0,0 +1,586 @@
+                            >
+                            <source
+                                media="(max-width: 1024px) and (min-width: 375px)"
+                                srcset="./src/images/work/work-6-452.webp"
+                                type="image/webp"
+                            >
+                            <source
+                                media="(max-width: 375px)"
+                                srcset="./src/images/work/work-6-256.webp"
+                                type="image/webp"
+                            >
+                            <source
+                                media="(max-width: 1024px and min-width: 375px)"
+                                srcset="./src/images/work/work-6-452.jpg"
+                            >
+                            <source
+                                media="(max-width: 375px)"
+                                srcset="./src/images/work/work-6-256.jpg"
+                            >
+                            <img loading="lazy" class="examples__image" src="./src/images/work/work-6-370.jpg" width="370" height="360" alt="Work 6">
+                        </picture>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="team">
+            <div class="team__container container">
+                <h2 class="team__title title">
+                    Small team
+                </h2>
+                <p class="team__title-addition title-addition">
+                    Lorem ipsum dolor sit amet event landing template
+                </p>
+                <ul class="team__list team-list">
+                    <li class="team-list__item">
+                        <picture class="team-list__picture">
+                            <source
+                                srcset="./src/images/team/team-1.webp"
+                                type="image/webp"
+                            >
+                            <img loading="lazy" class="team-list__image" src="./src/images/team/team-1.jpg" width="370" height="440" alt="Team chamber">
+                        </picture>
+                        <div class="team-list__info">
+                            <h3 class="team-list__title">
+                                AL RAYHAN <span class="team-list__title-addition">/ UI Designer</span>
+                            </h3>
+                            <p class="team-list__text">
+                                Lorem Ipsum is not simply is  an action designer random text
+It has roots in a piece. 
+                            </p>
+                            <ul class="team-list__social social1">
+                                <li class="social1__item">
+                                    <a href="#" class="social1__link">
+                                        <img loading="lazy" class="social1__icon" src="./src/images/team/team-facebook.png" alt="Social icon">
```

Comment:
use `.svg` format of icons here and beyond.

Reply:
I change img-files on svg.
```suggestion
                                        <img loading="lazy" class="social1__icon" src="./src/images/team/team-facebook.svg" alt="Social icon">
```